### PR TITLE
LEARNER-4694 Test - Android - Automate new Landing Screen

### PR DESCRIPTION
**Pull request by nedo**
Following scenarios are covered,

* Verify that Landing screen is loaded successfully
* Verify that Landing screen has following contents,
** edX logo
** Description text-field
** Search Courses edit-field
** Register button
** Sign In button
* Verify that all elements have their default values
* Verify that tapping "Log In" loads Log In screen
* Verify that tapping back icon from "Log In" screen navigate user back to Landing screen
* Verify that tapping "Create Your Account" loads Register screen
* Verify that tapping back icon from "Register" screen navigate user back to Landing screen
* Verify that tapping search after entering <course name> should load "Discovery" screen
* Verify that Discovery screen is showing
* Verify tapping back icon from 'Discovery' screen navigate user back to Landing screen

### CODE REVIEW

**Comment by nedo** [comment]
@raeeschachar PR is ready for review

cc @staubina

File: android/pages/android_new_landing.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        self.global_contents.flag = False
+        if self.driver.current_activity == self.global_contents.DISCOVERY_LAUNCH_ACTIVITY_NAME:
+            self.load_register_screen()
+
+            if self.driver.current_activity == self.global_contents.REGISTER_ACTIVITY_NAME:
+                self.global_contents.flag = True
+                self.driver.back()
+            else:
+                self.log.error('Register screen is not loaded')
+                self.global_contents.flag = False
```
Comment (raeeschachar) [inline android/pages/android_new_landing.py]: I don't think we need to set `self.global_contents.flag` to `False` here. As it is already set as False on line 185.

File: android/pages/android_new_landing.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        else:
+            self.log.error('Problem - Not on New Landing screen')
+            self.global_contents.flag = False
```
Comment (raeeschachar) [inline android/pages/android_new_landing.py]: Same here and same for `def back_and_forth_login(self):` method on line 153

File: android/pages/android_new_landing.py (hunk 1 above)
Comment (nedo) [reply android/pages/android_new_landing.py]: done

File: android/pages/android_new_landing.py (hunk 2 above)
Comment (nedo) [reply android/pages/android_new_landing.py]: done

**Comment by nedo** [comment]
@raeeschachar PR is ready for review

cc @staubina

**Comment by nedo** [comment]
@staubina PR is ready for review.

**Pull request merged**